Makefile

```diff
@@ -0,0 +1,76 @@
+# Copyright (C) Greenbone Networks GmbH
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+# Build and development command for the scan management backend project.
+# important commands are documented and machine-runnable at the same time here.
+
+PROJECT := go-project-template
+GOPATH := $(shell go env GOPATH)
+
+.PHONY: help
+# Show this help
+help: # So I managed to get the help printing with the commment line above it
```

Comment:
We might want to change that comment into something what the help target does now - the future readers and users will notice that someone made this work by using it ;) 

e.g. "This help function show the comments above each target as usage help"